dbgpt/model/proxy/llms/zhipu.py

```diff
@@ -84,26 +84,25 @@ def sync_generate_stream(
         request: ModelRequest,
         message_converter: Optional[MessageConverter] = None,
     ) -> Iterator[ModelOutput]:
-        import zhipuai
 
         request = self.local_covert_message(request, message_converter)
 
         messages = request.to_common_messages(support_system_role=False)
 
         model = request.model or self._model
         try:
-            res = zhipuai.model_api.sse_invoke(
+            response = self.client.chat.completions.create(
```

Comment:
The old version of`zhipuai` doesn't work properly. So I suggest checking the version here and reporting an error to remind users to upgrade it.

Reply:
Reminder has been added.